### ISSUE TITLE
An error occurred after the cluster started SSL for a period of time

### ISSUE THREAD

**Issue by Estheriii** [emqx/emqx#11518]
### What happened?

Building a 3+6 node cluster through an operator in k8s
Configure SSL listener
After running normally for a period of time, already build connection 200k , and an error occurred after 35 days

### What did you expect to happen?

no error

### How can we reproduce it (as minimally and precisely as possible)?

After running for a period of time, an error occurred, and every time the connection reached over 30000, it could not rise

### Anything else we need to know?

add ssl listeners
<img width="410" alt="image" src="https://github.com/emqx/emqx/assets/32813533/99440e89-1e9f-4ac2-97a2-b218cbeb39b1">
<img width="409" alt="image" src="https://github.com/emqx/emqx/assets/32813533/d602b141-da94-40d2-99ff-028a904a7751">
<img width="415" alt="image" src="https://github.com/emqx/emqx/assets/32813533/bd326fc2-4139-4952-a686-25ca25206173">

### EMQX version

<details>
[5.0.21-gd9d19ac3](https://www.emqx.com/en/changelogs/broker/5.0.21)
```console
$ ./bin/emqx_ctl broker
# paste output here
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Log files

<details>
`[error] supervisor: ‘esockd_connection_sup - <0.10480.11>’, errorContext: connection_shutdown, reason: {ssl_error,{tls_alert,{insufficient_security,“TLS server: In state hello at tls_handshake.erl:346 generated SERVER ALERT: Fatal - Insufficient Security\n no_suitable_ciphers”}}}, offender: [{pid,<0.12989.16>},{name,connection},{mfargs,{emqx_connection,start_link,[#{enable_authn => true,limiter => #{bytes_in => #{capacity => 1099511627776,initial => 0,rate => infinity},client => #{bytes_in => #{capacity => 1099511627776,divisible => false,failure_strategy => force,initial => 0,low_watermark => 0,max_retry_time => 10000,rate => infinity},connection => #{capacity => 1099511627776,divisible => false,failure_strategy => force,initial => 0,low_watermark => 0,max_retry_time => 10000,rate => infinity},message_in => #{capacity => 1099511627776,divisible => false,failure_strategy => force,initial => 0,low_watermark => 0,max_retry_time => 10000,rate => infinity},message_routing => #{capacity => 1099511627776,divisible => false,failure_strategy => force,initial => 0,low_watermark => 0,max_retry_time => 10000,rate => infinity}},connection => #{capacity => 1000,initial => 0,rate => 100.0},message_in => #{capacity => 1099511627776,initial => 0,rate => infinity},message_routing => #{capacity => 1099511627776,initial => 0,rate => infinity}},listener => {ssl,sslListener},zone => default}]}}]`
</details>

**Comment by kjellwinblad** [comment]
The error message provides a clue about the underlying problem. Specifically, it says:

`{ssl_error,{tls_alert,{insufficient_security,“TLS server: In state hello at tls_handshake.erl:346 generated SERVER ALERT: Fatal - Insufficient Security\n no_suitable_ciphers”}}}`

Here's a breakdown:

1. **insufficient_security**: This indicates that during the TLS handshake, the server determined that the security of the negotiated parameters is not sufficient to proceed with the handshake.

2. **no_suitable_ciphers**: This indicates that the server and client could not agree on a cipher suite, which is essential for encrypting the connection.

Now, let's attempt to deduce the possible reasons for the problem:

1. **Cipher Suite Mismatch**: The client may not be offering, or the server might not be supporting, the required cipher suites to establish a secure connection. This could be due to either party being configured with a limited set of ciphers or outdated ciphers.

2. **Outdated TLS Version**: The client may be attempting to use an outdated version of TLS, or the server might be set up to accept only the newer versions.

To summarize I think it seems like a new type of client that does not support any of the encryption methods that EMQX's TLS listener supports connected and that triggered the error log. That this happens after 30 days might just be a coincidence .

Do you know what type of software your clients are using? Could it be that a new type of client software tried to connect after 30 days that does not support the right types of ciphers?

Another random idea I have is that this might be related to that your TLS certificate has expired (this could explain why the error was only triggered after 30 days). So maybe you can check your certificates expiry day just to make sure.

Hope this helps. Please let us know if you think the problem is something else.

**Comment by Estheriii** [comment]
> 

We found that there should be a configuration issue with the load balancer tls policy
